FirebaseUI/Auth/AuthProviderUI/Google/Source/FIRGoogleAuthUI.m

```diff
@@ -31,10 +31,15 @@
  */
 static NSString *const kGooglePlusMeScope = @"https://www.googleapis.com/auth/plus.me";
 
-/** @var kGooglePlusScopesPrefix
-    @brief The OAuth scope string prefix for all scopes starting with "plus.".
+/** @var kGooglePlusMeScope
+ @brief The OAuth scope string for the user's email scope.
+ */
+static NSString *const kGoogleUserInfoEmailScope = @"https://www.googleapis.com/auth/userinfo.email";
+
+/** @var kGooglePlusMeScope
+ @brief The OAuth scope string for the basic G+ profile information scope.
  */
-static NSString *const kGooglePlusScopesPrefix = @"https://www.googleapis.com/auth/plus.";
+static NSString *const kGoogleUserInfoProfileScope = @"https://www.googleapis.com/auth/userinfo.profile";
```

Comment:
This one is a behavior change I think, though the effective difference between those scopes is slight, an upgrade may cause a reconsent. At the least we should flag this in changelog.